internal/hcsoci/create.go

```diff
@@ -229,3 +229,31 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	}
 	return system, resources, nil
 }
+
+func (coi *createOptionsInternal) isV2Xenon() bool {
+	if schemaversion.IsV21(coi.actualSchemaVersion) && coi.HostingSystem != nil {
```

Comment:
Couldn't all of these be shortened to 
```go
return schemaversion.IsV21(coi.actualSchemaVersion) && coi.HostingSystem != nil 
```
although line length might get a little long with this.

Reply:
Fair enough